services/ir1/alphabet/az.json

```diff
@@ -1,13 +1,12 @@
 {
-  "version": "3.0",
-  "name": null,
+  "version": "1.0",
   "accounts": [
     {
-      "address": "Nhfg3TbpwogLvDGVvAvqyThbsHgoSUKwtn",
-      "key": "6PYM8VdX2BSm7BSXKzV4Fz6S3R9cDLLWNrD9nMjxW352jEv3fsC8N3wNLY",
+      "address": "NZBEUULRXo79EqBQVPpUTziicepFc7kuXU",
```

Comment:
Looks like you're changing the FS chain CN, while I'd rather change the main chain CN wallet. FS chain is used more often in various scenarios, so keeping the address stable can be somewhat important.